app/views/microsoft/identity_association.json.erb

```diff
@@ -1,7 +1 @@
-{
```

Comment:
Can we use a JBuilder template instead?
To stay consistent with what we have in the codebase. 

lets also update the specs to verify the response headers.